www/viewgame

```diff
@@ -1455,7 +1452,13 @@ function submitTime() {
         document.getElementById("timeSaveMsg").innerHTML="The total time must be at least one minute.";
         return;
     }
-    // Time is valid, as far as we can tell
+    // Time is valid, as far as we can tell.
+    // If the game isn't already on the user's list of played games, add it.
+    if (document.getElementById('ckBoxckPlayed').checked==false) {
```

Comment:
put `document.getElementById('ckBoxckPlayed')` in a variable since it's used twice?

Also, instead of `thing.checked == false` you can do `!thing.checked`